### PR TITLE
Update vendor.md

### PR DESCRIPTION
**Pull request by akw28888**
Add some vendor ids

### CODE REVIEW

**Review by stich86 — COMMENTED** [review]
Looks good for me

**Review by stich86 — APPROVED** [review]

**Comment by simonebortolin** [comment]
What do you mean by external?

File: _gpon/vendor.md [hunk 1]
```diff
@@ -16,20 +16,26 @@ Here is a list of the most popular Vendor IDs:
...
 | `CDKT` | `43444B54` | KingType             |
 | `CIGG` | `43494747` | Cig                  |
 | `DLNK` | `444c4e4b` | Dlink                |
+| `DSNW` | `44534e57` | DASAN                |
 | `ELTX` | `454c5458` | Eltex                |
 | `FHTT` | `46485454` | Fiber Home           |
+| `GMTK` | `474d544b` | GemTek               |
 | `GNXS` | `474e5853` | Genexis              |
 | `GPON` | `47504f4e` | Generic vendor name  |
+| `GTHG` | `47544847` | Alcatel-Lucent (ODM) |
```
Comment (simonebortolin) [inline _gpon/vendor.md:29]: GemTek ODM?

File: _gpon/vendor.md [hunk 2]
```diff
@@ -38,8 +44,10 @@ Here is a list of the most popular Vendor IDs:
 | `TMBB` | `544d4242` | Technicolor          |
 | `TPLG` | `54504c47` | TP-Link              |
 | `UBNT` | `55424e54` | Ubiquiti             |
+| `UGRD` | `55475244` | UGrid                |
 | `YHTC` | `59485443` | Youhua               |
 | `ZTEG` | `5a544547` | ZTE                  |
 | `ZYWN` | `5a59574e` | Zyxel                |
+| `ZYXE` | `5a595845` | Zyxel                |
```
Comment (simonebortolin) [inline _gpon/vendor.md:51]: it should be specified which of the two zyxels is more common and/or where it is used (Example: EMEA ZYWN and America ZYXE, etc.)

File: _gpon/vendor.md (hunk 1 above)
Comment (prime-noc) [reply _gpon/vendor.md:29]: Yes, GemTek OEM

Comment (akw28888) [reply _gpon/vendor.md:29]: Yes.

File: _gpon/vendor.md (hunk 2 above)
Comment (akw28888) [reply _gpon/vendor.md:51]: I don't know which is more common, but they use ZYXE in some markets.
I have ZyXEL PMG4506 / 512NP, they are using ZYXE.

**Comment by akw28888** [comment]
> What do you mean by ODM in this case? Rebrand from GemTek?

These ONUs are made by GemTek, but branded as Alcatel/Nokia.
As I checked, it seems that Nokia is not involved in firmware development.
The HW/SW version doesn't look like the regular Nokia style(3FEXXXXXXXXX).

File: _gpon/vendor.md (hunk 1 above)
Comment (simonebortolin) [reply _gpon/vendor.md:29]: Please fix

**Comment by simonebortolin** [comment]
From https://hack-gpon.org/assets/img/ont-nokia-naming.jpg the namimg style are respected?

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**